frontend/packages/console-app/src/components/nodes/NodeTerminal.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import { Firehose, FirehoseResult, LoadingBox } from '@console/internal/components/utils';
+import { NodeKind, PodKind } from '@console/internal/module/k8s';
+import { PodExecLoader } from '../../../../../public/components/pod';
+import { PodModel } from '../../../../../public/models';
+import { k8sCreate, k8sKill } from '../../../../../public/module/k8s';
+
+type NodeTerminalInnerProps = {
+  obj?: FirehoseResult<PodKind>;
+};
+
+type NodeTerminalProps = {
+  obj: NodeKind;
+};
+
+const getDebugPod = (node) => {
+  return {
+    kind: 'Pod',
+    apiVersion: 'v1',
+    metadata: {
+      generateName: `${node.metadata.name}-debug-`,
+      namespace: 'default',
+      creationTimestamp: null,
+      annotations: {
+        'debug.openshift.io/source-container': 'container-00',
+        'debug.openshift.io/source-resource': `/v1, Resource=nodes/${node.metadata.name}`,
+      },
+    },
+    spec: {
+      volumes: [
+        {
+          name: 'host',
+          hostPath: {
+            path: '/',
+            type: 'Directory',
+          },
+        },
+      ],
+      containers: [
+        {
+          name: 'container-00',
+          image: 'registry.redhat.io/rhel7/support-tools',
```

Comment:
`oc` has some logic where it tries to get the image stream `openshift/tools:latest` and only falls back to this if unsuccessful. We should try to do the same.

https://github.com/openshift/oc/blob/6c8fa40ade687eec7a2d84e9780820961adfbcb2/pkg/cli/debug/debug.go#L804-L812